Stylesheet/Components/textbox.css

```diff
@@ -0,0 +1,137 @@
+/* This is textbox file */
+
+input {
+    outline: none;
```

Comment:
outline: none is not a good UX, instead you can try element: focus{ outline: none} this is what I found 
Removing outlines in CSS creates issues for people navigating the web with a keyboard.

Reply:
Okay then I will make it transparent